src/ostorlab/runtimes/local/runtime.py

```diff
@@ -170,6 +170,23 @@ def _docker_checks(self):
 
         self._docker_client = docker.from_env()
 
+    def prepare_scan(
```

Comment:
As discussed, the API should return once the scan is persisted the scan runtime should start in a dedicated thread or process.